README.md

```diff
@@ -350,3 +351,18 @@ $ wdl parse examples/ex2.wdl
         (RawCommand:
 ...
 ```
+
+# Converting To DOT
+
+A wdl file can be converted to the dot format in order to be able to visualize the pipeline as a graph. For example:
+
+```
+$ python scripts/wdl2dot.py -i hello.wdl -o hello.dot
```

Comment:
Can you rename `scripts/wdl2dot.py` to just `scripts/wdl2dot`, then do a `chmod +x` on it?

Then, in the `setup.py` file, add it to the scripts section:

```
scripts={
   'scripts/wdl',
   'scripts/wdl2dot'
}
```

This way, this line of the documentation can just be:

```
wdl2dot -i hello.wdl -o hello.dot
```